luma/lcd/device.py

```diff
@@ -649,6 +649,168 @@ def command(self, cmd, *args):
+    :param rotate: An integer value of 0 (default), 1, 2 or 3 only, where 0 is
+        no rotation, 1 is rotate 90° clockwise, 2 is 180° rotation and 3
+        represents 270° rotation.
+    :type rotate: int
+    :param framebuffer: Framebuffering strategy, currently instances of
+        ``diff_to_previous()`` or ``full_frame()`` are only supported.
+    :type framebuffer: luma.core.framebuffer.framebuffer
+    :param bgr: Set to ``True`` if device pixels are BGR order (rather than RGB).
+    :type bgr: bool
+    :param h_offset: Horizontal offset (in pixels) of screen to device memory
+        (default: 0).
+    :type h_offset: int
+    :param v_offset: Vertical offset (in pixels) of screen to device memory
+        (default: 0).
+    :type v_offset: int
+
+    .. versionadded:: ???
+    """
+    def __init__(self, serial_interface=None, width=320, height=480, rotate=0,
+                 framebuffer=None, h_offset=0, v_offset=0, bgr=False, **kwargs):
+        super(ili9486, self).__init__(luma.lcd.const.ili9486, serial_interface, **kwargs)
+        self.capabilities(width, height, rotate, mode="RGB")
+        self.init_framebuffer(framebuffer, 25)
+
+        if h_offset != 0 or v_offset != 0:
+            def offset(bbox):
+                left, top, right, bottom = bbox
+                return (left + h_offset, top + v_offset, right + h_offset, bottom + v_offset)
+            self.apply_offsets = offset
+        else:
+            self.apply_offsets = lambda bbox: bbox
+
+        # Supported modes
+        supported = (width, height) in [(320, 480)]  # full
+        if not supported:
+            raise luma.core.error.DeviceDisplayModeError(
+                "Unsupported display mode: {0} x {1}".format(width, height))
+
+        # RGB or BGR order
+        order = 0x00 if bgr else 0x08
+
+        # Initialization sequence, adapted from MIT Licensed
+        #
+        #   `https://github.com/juj/fbcp-ili9341/blob/master/ili9486.cpp`
+        #
+        # and peer files.  The sequence targets the ILI9486-based
+        # Waveshare "Wavepear" 3.5 inch 320x480 LCD (B)
+        # implementation.  Per comments in the above file and issue
+        # discussion for juj/fbcp-ili9341, the Waveshare
+        # implementation makes use of 16-bit shift register for the
+        # SPI interface, leaving the ILI9486 itself in a parallel
+        # mode.
+        #
+        # The result of that implementation is that registers
+        # effectively become 16-bit quantities.  The sequence below
+        # (and in the display() function) thus ends up padding
+        # commands and subsequent values.  It doesn't seem to have
+        # affected pixel transfers, though.
+        #
+        # A different ILI9486 implementation may NOT need the padding
+        # zeros.  The juj/fbcp-ili9341 code handles that possibility
+        # via a DISPLAY_SPI_BUS_IS_16BITS_WIDE ifdef.
+        #
+        # Yet another initialization sequence can be found at
```

Comment:
I wouldn't add this link, people can google for it.

Reply:
Do we need to leave any reminder in place, though, regarding the possible change for other ILI9486-based implementations?

Reply:
Oh, maybe you were referring just to the second link.  I'll remove that.

Reply:
yep, i was